src/workerd/server/workerd-api.c++

```diff
@@ -629,26 +629,28 @@ static v8::Local<v8::Value> createBindingValue(
 void WorkerdApiIsolate::compileGlobals(
     jsg::Lock& lockParam, kj::ArrayPtr<const Global> globals,
     v8::Local<v8::Object> target,
+    kj::Maybe<v8::Local<v8::Object>> processEnv,
     uint32_t ownerId) const {
   auto& lock = kj::downcast<JsgWorkerdIsolate::Lock>(lockParam);
-  v8::HandleScope scope(lock.v8Isolate);
+  auto isolate = lock.v8Isolate;
+  v8::HandleScope scope(isolate);
   auto context = lock.v8Context();
   auto& featureFlags = *impl->features;
 
   for (auto& global: globals) {
-    v8::HandleScope scope(lock.v8Isolate);
```

Comment:
A couple drive-by improvements made while I was in here. There's no reason to enter a new v8::HandleScope on each iteration since we already entered one at the top of the loop.